libraries/fiff/fiff_coord_trans.cpp

```diff
@@ -162,6 +162,25 @@ bool FiffCoordTrans::read(QIODevice& p_IODevice, FiffCoordTrans& p_Trans)
 
 //=============================================================================================================
 
+void FiffCoordTrans::write(QIODevice &qIODevice)
+{
+    // Create the file and save the essentials
+    FiffStream::SPtr pStream = FiffStream::start_file(qIODevice);
+    printf("Write coordinate transform in %s...\n", pStream->streamName().toUtf8().constData());
+    this->writeToStream(pStream.data());
+    qIODevice.close();
+}
+
+//=============================================================================================================
+
+void FiffCoordTrans::writeToStream(FiffStream* pStream)
+{
+    pStream->write_coord_trans(*this);
+    pStream->end_file();
```

Comment:
`pStream->end_file();` should be moved to the caller function.

Reply:
sure? I think I've seen this quite often in writeToStream functions.

Reply:
I think it should be changed in the other functions as well then. The name `writeToStream` does not imply that the stream is closed after we have written to it. This can be confusing from a dev perspective. What if at some point we want to write other things to stream as well. It would already have been closed.

Reply:
Yes, makes sence to me. Do w have to close the QIODevice within the write() function once writing is finished?